ext/include/opentelemetry/ext/http/client/curl/http_client_curl.h

```diff
@@ -259,6 +260,19 @@ class HttpClient : public http_client::HttpClient
     return session;
   }
 
+  std::shared_ptr<http_client::Session> CreateSession(nostd::string_view url) noexcept override
+  {
+    auto parsedUrl = common::UrlParser(std::string(url));
+    if (!parsedUrl.success_) {
+      return std::make_shared<Session>(*this, "empty-host", 80);
```

Comment:
Should `empty-host` actually be an empty string?  `empty-host` is a valid DNS name and could indeed be the hostname targeted, or it could have something entirely different listening at port 80.

Reply:
@Aneurysm9 I definitely agree it should be an empty string, it was originally something the creator of the issue told me to return when I asked what should happen if it fails to parse. I'll set it to an empty string it seems like a more safe option. The idea is is that we let the actual connection failure get caught at ResponseHandler level.